src/ui/components/LedgerAdmin.js

```diff
@@ -56,6 +73,9 @@ export const LedgerAdmin = (): ReactNode => {
   const [selectedId, setSelectedId] = useState<IdentityId | null>(null);
   const [promptString, setPromptString] = useState<string>("Add Identity:");
   const [checkboxSelected, setCheckBoxSelected] = useState<boolean>(false);
+  const filteredLedger = useTableState(
+    ledger.accounts().map((a) => a.identity)
```

Comment:
You should probably put this in a memo, as TableState does not do that for you. You can do this inline if you want.
```
useTableState(
    useMemo(ledger.accounts().map((a) => a.identity), [ledger])
)
```

Reply:
Got it, thanks!